### PR TITLE
b04 takes over b02

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/test-platform

### CODE REVIEW

File: core-services/sanitize-prow-jobs/_config.yaml [hunk 1]
```diff
@@ -5450,6 +5450,9 @@ groups:
     - openshift-openshift-azure-infra-periodics.yaml
     - openshift-release-infra-periodics.yaml
     - infra-periodics-origin-release-images.yaml
+  build01:
+    paths:
+    - .*-postsubmits.yaml$
```
Comment (petr-muller) [inline core-services/sanitize-prow-jobs/_config.yaml:5455]: this needs to go in the build01 list above, this is a duplicate key

Comment (hongkailiu) [reply core-services/sanitize-prow-jobs/_config.yaml:5455]: Stupid me!
On it.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**